app/src/main/java/com/nononsenseapps/feeder/ui/compose/settings/OpenAISection.kt

```diff
@@ -219,6 +228,34 @@ fun OpenAISectionEdit(
                 onEvent(OpenAISettingsEvent.UpdateSettings(current.copy(baseUrl = it)))
             },
         )
+
+        TextField(
+            modifier =
+                Modifier.fillMaxWidth().onFocusChanged {
+                    if (!(it.isFocused || isTimeoutInputValid)) {
+                        timeoutString = timeoutString.toInt().coerceIn(30, 600).toString()
+                    }
+                },
```

Comment:
```suggestion
                Modifier.fillMaxWidth(),
```

Wrong place for this. The onValueChange would have been better, but I see no reason to do any of that in the UI component since you have proper error checking here. But on the backend tough..

Reply:
I've moved the coerceIn call to settingsStore. The change did make the value that's storing into settingsStore correct.

However, when the user opens the openAI settings again, they will see the invalid value they had typed (and the error message), this will be confusing to the user but it is somewhat expected, cause the timeoutString was never updated to reflect the clamped value.

I also attempted the coerceIn call on timeoutString within onValueChange, but that will interfere with user input -- when they type 6, the input field will be concerce immdiately to 30, which is also not ideal. Would you have any other suggestions on this?

Reply:
> However, when the user opens the openAI settings again, they will see the invalid value they had typed (and the error message), this will be confusing to the user but it is somewhat expected, cause the timeoutString was never updated to reflect the clamped value.

I've added a comment which should address this. The value is never changed if it is not valid. Closing settings and coming back should load the stored value.